Unfolding/notebooks/nSubExtractor.py

```diff
@@ -0,0 +1,216 @@
+import ROOT
```

Comment:
do you need this python script here? I think a better place is under `Unfolding/python/`
As a general rule in CMSSW, small python scripts or configs to run with cmsRun are located in `test/`, all the rest python classes like this one (or the nsubproducer) are located in `python/`.

Reply:
Point taken, I've moved the script as suggested.